docs/eBpfExtensions.md

```diff
@@ -322,6 +322,17 @@ ebpf_extension_data_t* extension_data = (ebpf_extension_data_t*)ClientRegistrati
 attach_parameter = extension_data->data;
 ```
 
+Note: This structure can be either version EBPF_ATTACH_CLIENT_DATA_VERSION_1 or version EBPF_ATTACH_CLIENT_DATA_VERSION_2.
+It is strongly recommended that hook providers are able to support version 2 as version 1 will be eventually deprecated.
```

Comment:
```suggestion
It is strongly recommended that hook providers support version 2 as version 1 will be eventually deprecated.
```
Also, for the "will eventually be deprecated" point, is there an issue number?  Should they be deprecated now?